app/experimenter/static/core/package.json

```diff
@@ -29,7 +29,7 @@
   "description": "a tool for managing experiments in Firefox",
   "dependencies": {
     "@fortawesome/fontawesome-free": "5.14.0",
-    "@testing-library/dom": "latest",
```

Comment:
This was resulting in a yarn warning (I didn't copy it all):

```
Pattern ["@testing-library/dom@latest"] is trying to unpack in the same destination "/Users/jody/Library/Caches/Yarn/v6/npm-@testing-library-dom-7.24.2-6d2b7dd21efbd5358b98c2777fc47c252f3ae55e-integrity/node_modules/@testing-library/dom" as pattern ["@testing-library/dom@^7.24.2","@testing-library/dom@^7.24.2","@testing-library/dom@^7.17.1","@testing-library/d...
```

Using `latest` is probably not the best for stability.